resources/dist/linux/share/metainfo/org.moneymanagerex.MMEX.metainfo.xml

```diff
@@ -342,17 +342,19 @@
 
   <releases>
     
```

Comment:
@n-stein @vomikan This will need an entry for `v1.6.4.1` or `v1.6.5` as well before tagging a new release.

Reply:
Does the date string need to be correct?

Reply:
Reverted version to 1.6.4.1

Reply:
The date string should reflect the day that you planned to release on, but there aren't strict requirements.